src/phylo/phylogeny.jl

```diff
@@ -105,6 +104,11 @@ function Phylogeny{T <: AbstractString}(taxa::Vector{T})
     return Phylogeny(convert(Vector{Symbol}, taxa))
 end
 
+"""
+    Base.copy{C, B}(tree::Phylogeny{C, B})
+
+Create a copy of a phylogenetic tree.
+"""
```

Comment:
This docstring is redundant; this method does the same thing as `Base.copy`, which is already documented.